configure.ac

```
@@ -424,7 +424,8 @@ fi
 
 case $host_os in
   darwin*)
-    if ! test "`/usr/bin/sw_vers | grep ProductVersion: | cut -f 2 | cut -d. -f2`" -ge 7; then
+    if ! test "`/usr/bin/sw_vers | grep ProductVersion: | cut -f 2 | cut -d. -f2`" -ge 7 -o \
+       "`/usr/bin/sw_vers | grep ProductVersion: | cut -f 2 | cut -d. -f1`" -ge 10; then
```

This causes 10.6 and lower to be also accepted when they shouldn't.
```suggestion
       "`/usr/bin/sw_vers | grep ProductVersion: | cut -f 2 | cut -d. -f1`" -gt 10; then
```

---

Thank you for catching that!